src/app/report/edit-display/encoding-button/encoding-button.component.html

```diff
@@ -0,0 +1,3 @@
+@if (ReportUtil.isCheckPoint(selectedNode) && selectedNode.encoding === 'Base64') {
```

Comment:
This if statement can be simplified to a class variable that is updated in `ngOnChanges`

Reply:
added the change